extensions/python/types/PyProcessSession.h

```diff
@@ -39,6 +39,7 @@ class PyProcessSession {
   void remove(const std::shared_ptr<core::FlowFile>& flow_file);
   std::string getContentsAsString(const std::shared_ptr<core::FlowFile>& flow_file);
   void putAttribute(const std::shared_ptr<core::FlowFile>& flow_file, std::string_view key, const std::string& value);
+  gsl::not_null<core::ProcessSession*> getSession() const { return session_; }
```

Comment:
This should return a reference instead.

Reply:
Good point, updated in 1366b3b397a13faa54769543083ff9b46290ebf7